cob_hardware_config/cob3-2/config/teleop.yaml

```diff
@@ -9,6 +9,7 @@ arm_joint12_button: 0
 arm_joint34_button: 1
 arm_joint56_button: 2
 arm_joint7_button: 3
+base_safety_button: 4
```

Comment:
are you sure that the cob3-2 joystick version has this button and in this position?